components/docs-chef-io/content/automate/session_timeout.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Session Timeout"
+
+weight = 70
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Session Timeout"
+    parent = "automate/configuring_automate"
+    identifier = "automate/configuring_automate/session_timeout.md Session Timeout"
+    weight = 70
++++
+
+Chef Automate has an optional session timeout configuration for signing out idle workstations. Chef Automate measure session activity as any interaction from a workstation, from a mouse, keyboard, or touchpad. With session timeout enabled, Chef Automate signs out idle workstations after a set number of minutes.
+
+To enable session_timeout in Chef Automate:
+
+- Set the value of `enable_idle_timeout` property to **true** in *config.toml* file.
+- Set the value of `idle_time_minutes` property to your desired time in minutes(for example **30**). (Setting the value of this property to **30** will set the session timeout time to 30 minutes).
```

Comment:
It should be `idle_timeout_minutes`. 
The `idle_timeout_minutes` should be more than 5 minutes